### PR TITLE
Fixes #190

### LINKED ISSUE #190: Session Card On homepage not showing image.
Ideally, the session card on the home page should show an image. You can compare it with the image when you open Sessions on the bottom navigation and open any session.

Check the screenshot below.
image

### PR DESCRIPTION
**Pull request by githukueliud**
# Scope

_Please make sure to read the [Contribution Guidelines](https://github.com/droidconKE/droidconKE2023Android/blob/main/CONTRIBUTING.md)
and check that you understand and have followed it as best as possible Explain what your feature
does in a short paragraph._ please check the below boxes
- [x] I have followed the coding conventions
- [x] I have added/updated necessary tests
- [x] I have tested the changes added on a physical device
- [x] I have run `./codeAnalysis.sh` on linux/unix or `codeAnalysys.bat` on windows to make sure all lint/formatting checks have been done.

## Closes/Fixes Issues
_Declare any issues by typing `fixes #1` or `closes #1` for example so that the automation can kick
in when this is merged_

## Other testing QA Notes
_What have you tested specifically and what possible impacts/areas there are that may need retesting
by others._

Please add a screenshot (if necessary)

### CODE REVIEW

File: presentation/src/main/java/com/android254/presentation/home/components/HomeSessionSection.kt [hunk 1]
```diff
@@ -112,7 +117,8 @@ fun HomeSectionHeader(
                 text = stringResource(id = R.string.view_all_label),
                 textAlign = TextAlign.Start,
                 style = TextStyle(
-                    color = ChaiBlue,
+                    //made the color for the sessions dynamic for Dark mode visibility
+                    color = if (isSystemInDarkTheme()) Color.White else ChaiBlue,
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/home/components/HomeSessionSection.kt:121]: This is already handled in the `ChaiDCKE22Theme`. Check out `ChaiColors`, You can add an entry on  `ChaiLightColorPalette` and `ChaiDarkColorPalette` and use it in code like this `MaterialTheme.chaiColorsPalette.your-color`

File: presentation/src/main/java/com/android254/presentation/home/components/HomeSpeakersSection.kt [hunk 2]
```diff
@@ -111,7 +112,7 @@ fun HomeSpeakersSection(
                         text = "${speakers.size}",
                         modifier = Modifier.align(Alignment.Center),
                         style = TextStyle(
-                            color = colorResource(id = R.color.blue),
+                            color = if (isSystemInDarkTheme()) Color.White else colorResource(id = R.color.blue),
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/home/components/HomeSpeakersSection.kt:115]: same comment as above

File: presentation/src/main/res/drawable-nodpi/sessions_background.png
Comment (chepsi) [inline presentation/src/main/res/drawable-nodpi/sessions_background.png:1]: I don't think we needed to add this. the API provides a session image check out the session details page.

**Comment by githukueliud** [comment]
I think you mean like how we put color =
MaterialTheme.colorScheme.body_text this will be dynamic for the dark mode
and light mode. But in this case it has been hard coded ChaiBlue, I just
put an if condition to display white color on dark mode because ChaiBlue is
not visible in dark mode.

Is that what you mean?

File: presentation/src/main/res/drawable-nodpi/sessions_background.png
Comment (githukueliud) [reply presentation/src/main/res/drawable-nodpi/sessions_background.png:1]: Alright I saw it, the one used for the sessions is from droidcon berlin, I even had to edit, so as to place the session title and speaker details. 
image

Comment (chepsi) [reply presentation/src/main/res/drawable-nodpi/sessions_background.png:1]: That's just a design guide. the image comes from the API.

File: presentation/src/main/java/com/android254/presentation/home/components/HomeSessionSection.kt (hunk 1 above)
Comment (chepsi) [reply presentation/src/main/java/com/android254/presentation/home/components/HomeSessionSection.kt:121]: @githukueliud check this PR. https://github.com/droidconKE/droidconKeKotlin/pull/212 should give you an idea of what I mean.

Comment (githukueliud) [reply presentation/src/main/java/com/android254/presentation/home/components/HomeSessionSection.kt:121]: ok, on it

**Comment by chepsi** [comment]
@githukueliud marking this as stale and closing it for now.

**Pull request closed without merging**